### PR TITLE
[PORT] Floor Painter from para.

### PR DESCRIPTION
**Pull request by Moccha-Bee**
## About The Pull Request
Adds the floor painter from para with a better sprite made by myself
Credit goes to JupiteKat and Monster1860 from paradise who originally created this.
Original pr: https://github.com/austation/austation/pull/2157

image
image

## Why It's Good For The Game
I want to paint my goddam floors.

## Changelog
:cl:
add: Added Floor Painter from para.
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Does this include decals or just icon states for various turfs?

**Comment by Moccha-Bee** [comment]
> Does this include decals or just icon states for various turfs?

Icon state for turfs

**Comment by patpol4** [comment]
Aw, pretty cool anyway, might port this to yogs soon.

File: code/modules/research/designs/tool_designs.dm [hunk 1]
```diff
@@ -93,6 +93,16 @@
 	category = list("Tool Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
 
+/datum/design/floorpainter
```
Comment (qwertyquerty) [inline code/modules/research/designs/tool_designs.dm:96]: Why do we have two of these

**Comment by qwertyquerty** [comment]
It looks like you added a ton of icon states from para that are just unused in the codebase except for this. I'm fairly sure we use overlays for floors or something

**Comment by Moccha-Bee** [comment]
> It looks like you added a ton of icon states from para that are just unused in the codebase except for this. I'm fairly sure we use overlays for floors or something

Most of these floor icons are just the normal floors with decals on top of them,making them into custom floors. nothing from para itself.

File: code/modules/research/designs/tool_designs.dm (hunk 1 above)
Comment (Moccha-Bee) [reply code/modules/research/designs/tool_designs.dm:96]: one is for the autolathe and the other for the protolathe

**Pull request closed without merging**